.platform/covasim-service.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v1
+kind: Pod
```

Comment:
Unless we are running 1 node k8s, in which case we shouldn't run any k8s, this should be `Deployment` with several replicas

Reply:
This isn't the production version. I intended this for local testing. I will include the true production version as well which includes replicas. Should we have one for local testing then or just one for for both environments?

Reply:
I'd run just one. You can run deployments on microk8s/minikube no problem, even with replicas.

Reply:
I add replicas and also ingress files for azure and microk8s. @inc0 Please review. My kubernetes knowledge is probably not as deep as yours